src/main.ts

```diff
@@ -24,10 +27,37 @@ export async function run() {
       await cake.bootstrapScript(scriptPath, toolsDir);
     }
 
-    await cake.runScript(scriptPath, toolsDir, target, verbosity);
+    await cake.runScript(
+      scriptPath,
+      toolsDir,
+      target,
+      verbosity,
+      ...scriptArgs
+    );
   } catch (error) {
     core.setFailed(error.message);
   }
 }
 
+function getScriptArguments(): CakeArgument[] {
+  const result: CakeArgument[] = [];
+  const input: string = core.getInput("script-arguments");
+  if (!input) {
+    return [];
+  }
```

Comment:
You don't need this check. If the `script-arguments` parameter isn't passed to the action, [`core.getInput` returns an empty string](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L85-L86). And passing an empty string to `yargsParser` simply results in an empty array.

```suggestion
```